left_arm:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100  
  joint_trajectory_controller:
    type: position_controllers/JointTrajectoryController
    joints: 
      - left_arm_0_joint
      - left_arm_1_joint
      - left_arm_2_joint
      - left_arm_3_joint
      - left_arm_4_joint
      - left_arm_5_joint
      - left_arm_6_joint

  PotentialFieldControl:
    type: desperate_housewife/PotentialFieldControl
    root_name: vito_anchor
    tip_name: left_hand_palm_ref_link
    desired_reference_topic: desired_hand_left_pose
    obstacle_remove_topic: Reject_obstacle_list_left
    obstacle_avoidance: obstacle_pose_left
    set_gains_topic: set_gains_left
    time_interp_desired: 5
    percentage: 0.5
    repulsive_gains: 0.1
    repulsive_treshold: 1
    Repulsive_table: 0.15

left_hand:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100  
  joint_trajectory_controller:
    type: position_controllers/JointTrajectoryController
    joints: 
      - left_hand_synergy_joint

right_arm:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100  
  joint_trajectory_controller:
    type: position_controllers/JointTrajectoryController
    joints: 
      - right_arm_0_joint
      - right_arm_1_joint
      - right_arm_2_joint
      - right_arm_3_joint
      - right_arm_4_joint
      - right_arm_5_joint
      - right_arm_6_joint

  PotentialFieldControl:
    type: desperate_housewife/PotentialFieldControl
    root_name: vito_anchor
    tip_name: right_hand_palm_ref_link
    desired_reference_topic: desired_hand_right_pose
    obstacle_remove_topic: Reject_obstacle_list_right
    obstacle_avoidance: obstacle_pose_right
    set_gains_topic: set_gains_right
    time_interp_desired: 5
    percentage: 0.5
    repulsive_gains: 0.01
    repulsive_treshold: 1000
    Repulsive_table: 0.15

right_hand:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100  
  joint_trajectory_controller:
    type: position_controllers/JointTrajectoryController
    joints: 
      - right_hand_synergy_joint

head:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100  
  joint_trajectory_controller:
    type: position_controllers/JointTrajectoryController
    joints: 
      - head_neck_pitch_joint
      - head_neck_roll_joint
      - head_neck_yaw_joint
      - head_head_tilt_joint
      - head_eyes_tilt_joint
      - head_left_eye_joint
      - head_right_eye_joint

